addons/debugger/addon.json

```diff
@@ -24,7 +31,52 @@
       "matches": ["projects"]
     }
   ],
+  "settings": [
+    {
+      "name": "Clear logs on green flag",
+      "id": "log_clear_greenflag",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Log green flag clicks",
+      "id": "log_greenflag",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Log clone creation",
+      "id": "log_clone_create",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Log when max clones exceeded",
+      "id": "log_failed_clone_creation",
+      "type": "boolean",
+      "default": false
```

Comment:
@Tacodude7729 Any reason not to set the default to true for max clones?
Logging green flag by default would be annoying because there would be unread messages every time you click the green flag, but max clones is much less frequent and important.

Reply:
No there is no reason. I have enabled this one by default and the invalid cloud variable one too, given I think it's very rare somebody is setting invalid cloud data on purpose.